README.md

```diff
@@ -4,6 +4,8 @@
 
 ## Project Status
 
+***This project's code base is still under development and breaking changes will happen. Whilst the maintainers will do our best to minimise disruption to existing deployments, this may not always be possible. As the project becomes more mature we will start creating stable releases as and when appropriate***
```

Comment:
I think the final text should be "as and when" or "as appropriate" or "when appropriate", but not combined.

Personally, I think the best option is turn the sentence around...

`Stable releases will be published when the project is more mature."